src/user/data_facilities_and_queues.rst

```diff
@@ -15,7 +15,7 @@ In the table below, the minRSS and maxRSS means the range of required memory.
 that is held in main memory (RAM). BPS ``requestMemory`` will be mapped to
 ``RSS`` requirements in PanDA.
 For example, for a Rubin job with "requestMemory: 5000" (5000MB), PanDA will
-schedule the job to *<site>_Rubin_medium* which has minRSS 4GB and maxRSS 8GB.
+schedule the job to *<site>_Rubin_8G* which has minRSS 4GB and maxRSS 8GB.
```

Comment:
I know this isn't something new for this pull request, but this time I really noticed that the maxRSS of one queue matches the minRSS of the next queue and I didn't see in the docs (sorry if I just missed it) which queue a job exactly matching the boundary RSS would go (e.g., requestMemory of 8GB would go in the _8G queue or the _12G queue).

Reply:
In fact, in the CRIC definition, minRSS 4G means 4001, maxRSS 4G means 4000. So the next queue doesn't overlap the previous queues. I will add this part in the doc.